src/utils.ts

```diff
@@ -254,14 +254,16 @@ export async function multicall(
 }
 
 export async function subgraphRequest(url: string, query, options: any = {}) {
+  const body: Record<string, any> = { query: jsonToGraphQLQuery({ query }) };
+  if (body) body.variables = options.variables;
```

Comment:
I think body will always exist
```suggestion
  if (options.variables) body.variables = options.variables;
```